marshmallow/schema.py

```diff
@@ -540,13 +545,20 @@ def loads(self, json_data, many=None, *args, **kwargs):
         :param str json_data: A JSON string of the data to deserialize.
         :param bool many: Whether to deserialize `obj` as a collection. If `None`, the
             value for `self.many` is used.
+        :param bool partial: Whether to ignore missing fields. If `None`, the
+            value for `self.partial` is used.
         :return: A tuple of the form (``data``, ``errors``)
         :rtype: `UnmarshalResult`, a `collections.namedtuple`
 
         .. versionadded:: 1.0.0
         """
+        # TODO: This avoids breaking backward compatibility if people were
+        # passing in positional args after `many` for use by `json.loads`, but
+        # ideally we shouldn't have to do this.
+        partial = kwargs.pop('partial', None)
```

Comment:
This is ok for now. It doesn't significantly reduce readability. We can remove it in marshmallow 3.